### PR TITLE
update library-go and match manifests with kube-apiserver operator for serving metrics

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: manifests/0000_20_cluster-openshift-apiserver-operator_06_service.yaml [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
   ports:
-  - name: https
-    port: 443
+  - port: 443
+    protocol: TCP
     targetPort: 8443
+  selector:
+    app: openshift-cluster-openshift-apiserver-operator
+  sessionAffinity: None
+  type: ClusterIP
```
Comment (deads2k) [inline manifests/0000_20_cluster-openshift-apiserver-operator_06_service.yaml]: nit tomas likes trailing newlines

File: manifests/0000_20_cluster-openshift-apiserver-operator_06_service.yaml [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+spec:
   ports:
-  - name: https
-    port: 443
+  - port: 443
+    protocol: TCP
     targetPort: 8443
+  selector:
+    app: openshift-cluster-openshift-apiserver-operator
+  sessionAffinity: None
```
Comment (deads2k) [inline manifests/0000_20_cluster-openshift-apiserver-operator_06_service.yaml]: didn't expect this to be specified.

File: manifests/0000_20_cluster-openshift-apiserver-operator_07_deployment.yaml [hunk 3]
```diff
@@ -24,19 +24,20 @@ spec:
         command: ["cluster-openshift-apiserver-operator", "operator"]
         args:
         - "--config=/var/run/configmaps/config/config.yaml"
-        - "-v=4"
+        - "-v=2"
         volumeMounts:
         - mountPath: /var/run/configmaps/config
           name: config
-        - mountPath: /var/run/secrets/serving-cert
+        - mountPath: /var/run/secrets/serving-certs
```
Comment (deads2k) [inline manifests/0000_20_cluster-openshift-apiserver-operator_07_deployment.yaml]: pretty sure I didn't pluralize this in the defaults.  Did I?

Comment (deads2k) [reply manifests/0000_20_cluster-openshift-apiserver-operator_07_deployment.yaml]: > pretty sure I didn't pluralize this in the defaults. Did I?

yeah, not plural https://github.com/openshift/library-go/blob/master/pkg/config/configdefaults/config_default.go#L48

**Comment by mfojtik** [comment]
already fixed

**Pull request closed without merging**